src/webots/gui/WbGuiApplication.hpp

```diff
@@ -59,6 +59,7 @@ class WbGuiApplication : public QApplication {
   WbStreamingServer *mStreamingServer;
 
   void parseArguments();
+  void parseStreamArguments(const QString &);
```

Comment:
Usually we always also add the argument name:
```suggestion
  void parseStreamArguments(const QString &serverArguments);
```

not sure why this is not the case on line 66.